<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Shifts</class>
 <widget class="QWidget" name="Shifts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Shifts</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="treeView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Shift Details</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Shift Name:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="5">
         <widget class="QLineEdit" name="ShiftName"/>
        </item>
        <item row="0" column="6">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Description:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="7" colspan="2">
         <widget class="QLineEdit" name="Description"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Start Time:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QTimeEdit" name="StartTime">
          <property name="time">
           <time>
            <hour>8</hour>
            <minute>0</minute>
            <second>0</second>
           </time>
          </property>
          <property name="displayFormat">
           <string>hh:mm:ss</string>
          </property>
         </widget>
        </item>
        <item row="1" column="6">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>End Time:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="7">
         <widget class="QTimeEdit" name="EndTime">
          <property name="time">
           <time>
            <hour>8</hour>
            <minute>0</minute>
            <second>0</second>
           </time>
          </property>
          <property name="displayFormat">
           <string>hh:mm:ss</string>
          </property>
         </widget>
        </item>
        <item row="2" column="7">
         <widget class="QTimeEdit" name="OvertimeStart">
          <property name="time">
           <time>
            <hour>8</hour>
            <minute>0</minute>
            <second>0</second>
           </time>
          </property>
          <property name="displayFormat">
           <string>hh:mm:ss</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Maximum Overtime Minutes:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" colspan="2">
         <widget class="QSpinBox" name="MaximumOtMinutes"/>
        </item>
        <item row="3" column="3" colspan="3">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Lunch Minutes:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="6">
         <widget class="QSpinBox" name="LunchMinutes"/>
        </item>
        <item row="3" column="7">
         <widget class="QCheckBox" name="HasPenalties">
          <property name="text">
           <string>Has Penalties</string>
          </property>
         </widget>
        </item>
        <item row="2" column="6">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Overtime Start:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="2">
         <widget class="QCheckBox" name="HasOvertime">
          <property name="text">
           <string>Has Overtime</string>
          </property>
         </widget>
        </item>
        <item row="1" column="8">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="9">
         <widget class="QCheckBox" name="Monday">
          <property name="text">
           <string>Monday</string>
          </property>
         </widget>
        </item>
        <item row="1" column="9">
         <widget class="QCheckBox" name="Tuesday">
          <property name="text">
           <string>Tuesday</string>
          </property>
         </widget>
        </item>
        <item row="2" column="9">
         <widget class="QCheckBox" name="Wednesday">
          <property name="text">
           <string>Wednesday</string>
          </property>
         </widget>
        </item>
        <item row="3" column="9">
         <widget class="QCheckBox" name="Thursday">
          <property name="text">
           <string>Thursday</string>
          </property>
         </widget>
        </item>
        <item row="0" column="10">
         <widget class="QCheckBox" name="Friday">
          <property name="text">
           <string>Friday</string>
          </property>
         </widget>
        </item>
        <item row="1" column="10">
         <widget class="QCheckBox" name="Saturday">
          <property name="text">
           <string>Saturday</string>
          </property>
         </widget>
        </item>
        <item row="2" column="10">
         <widget class="QCheckBox" name="Sunday">
          <property name="text">
           <string>Sunday</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="bottomMargin">
        <number>50</number>
       </property>
       <item>
        <widget class="QPushButton" name="SaveAndNew">
         <property name="text">
          <string>Save And New</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Save">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Delete">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
